src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobResumeBuild.java

```diff
@@ -0,0 +1,45 @@
+package com.tikal.jenkins.plugins.multijob;
+
+import hudson.model.AbstractBuild;
+import hudson.model.AbstractProject;
+import hudson.model.Cause;
+import hudson.model.ProminentProjectAction;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+import java.io.IOException;
+
+public class MultiJobResumeBuild implements ProminentProjectAction {
+
+	private final AbstractBuild<?, ?> build;
```

Comment:
:bug: As I see, this action is being persisted to the disk, In such case usage of the non-transient <code>build</code> field may lead to serialization issues leading to the data duplication and even recursions. It should not happen in general, but it's a bad practice.

I'd recommend to implement <code>RunAction2</code> interface and convert the field to the transient one.